src/android/DatagramSocketListener.java

```diff
@@ -49,6 +49,10 @@ public void run() {
                 Log.d(Dgram.TAG, "Received message " + message + " from " + address + " and port " + port);
                 emitMessageResult(message, address, port);
             } catch (Exception e) {
+                if (this.isInterrupted()) {
```

Comment:
again - calling methods on yourself is typically referenced just using the method name